chart/values.yaml

```diff
@@ -0,0 +1,160 @@
+# This values.yaml file provides default configuration for Turing Helm chart. 
+# It is also currently used to generate README.md using helm-docs
+# https://github.com/norwoodj/helm-docs
+
+turing:
+  image:
+    # -- Docker registry for Turing API image. User is required to override
+    # the registry for now as there is no publicly available Turing image
+    registry: docker.io/
+    # -- Docker image repository for Turing API
+    repository: turing
```

Comment:
Doesn't seem crucial for now, cos we don't publish images yet, but I guess it's going to be `gojek/turing`.
Also, we should consider using ghcr.io as Merlin doesn, instead of docker.io

Reply:
Yeah sure, I'm planning to use GitHub container registry as well.